src/design/colors.stories.mdx

```diff
@@ -125,3 +125,20 @@ We design and build with accessibility in mind. Color contrast for text should m
     colors={[colors.text.action.value]}
   />
 </ColorPalette>
+
+## Status Colors
+
+Includes colors that can reflect the status of elements.
+
+<ColorPalette>
+  <ColorItem
+    title="$status-error-dark"
+    subtitle="For errors, intended to be used for the foreground"
```

Comment:
These subtitles should be `comment` values in the design tokens so they are more easily accessible in the future.